tools/github-event-processor/Azure.Sdk.Tools.GitHubEventProcessor/Utils/IssueTriageOutput.cs

```diff
@@ -0,0 +1,25 @@
+using Octokit;
+using System;
+using System.Linq;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Azure.Sdk.Tools.GitHubEventProcessor.Utils
+{
+    /// <summary>
+    /// Class used to deserialize AI Triage output.
+    /// Has four states:
+    /// - Empty: No labels, no answer, no answer type
+    /// - Labels: Labels are provided, no answer, no answer type. 
+    /// - Suggestion: Labels are provided, answer is provided, answer type = "suggestion".
+    /// - Solution: Labels are provided, answer is provided, answer type = "solution".
+    /// </summary>
+    public class IssueTriageOutput
```

Comment:
```suggestion
    public class IssueTriageResponse
```

another really nitty thing.   Generally the name "Response" is preferred in across the .NET packages for "the service returned this thinger."    It doesn't really matter here, as this is internal, but felt worth mentioning.

I leave it to your discretion whether to rename or ignore me.   I'm good either way.  😄